### ISSUE TITLE
Insufficient contrast ratio between background and foreground colors on "Tags -> Create new tag" page

### ISSUE THREAD

**Issue by abbyli437** [CMU-313/Mayan-EDMS#25]
The current Lighthouse accessibility score is 85. The Lighthouse-reported problem being targeted is that the color of the text "A short text used as the tag name", "The RGB color values for the tag", and the Cancel button do not have a sufficient contrast ratio with the color of the background. This lack of contrast makes it hard for users to read.
